lib/pages/new_group/new_group.dart

```diff
@@ -270,12 +293,35 @@ class NewGroupController extends ContactsSelectionController<NewGroup>
     context.go("/rooms/$roomId");
   }
 
+  void _getImageOnWeb(
+    BuildContext context,
+  ) async {
+    final result = await FilePicker.platform.pickFiles(
+      type: FileType.image,
+    );
+    Logs().d(
+      'NewGroupController::_getImageOnWeb(): FilePickerResult - $result',
+    );
```

Comment:
imo log shouldn't be before null checking wdyt ?

Reply:
no problem